internal/display/display.go

```diff
@@ -272,9 +273,21 @@ func OutputObject(value map[string]any, serviceName, templateContent string, out
 			exitError("failed to execute template: %s", err)
 		}
 
+		// Define word wrap for the renderer.
+		// Use 80 characters by default, or the terminal width if available.
+		wordWrap := 80
+		termFd := os.Stdout.Fd()
+		if term.IsTerminal(termFd) {
+			termWidth, _, _ := term.GetSize(os.Stdout.Fd())
```

Comment:
style: could be more compact
```suggestion
			if termWidth, _, _ := term.GetSize(termFd); termWidth > 0 {
```

Reply:
Good point, I also missed using termFd here. I've changed that.